Node-DC-EIS-client/util.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2016 Intel Corporation 
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+import os
+start_MT = 0 
+end_MT = 0
+MT_req = 0
+
+def record_start_time():
+  """
+  # Desc  : Function to record start time
+  # Input : None
+  # Output: Returns start time of measuring time phase
+  """
+  global start_MT
+  start_MT = time.time()
+  
+
+def record_end_time():
+  """
+  # Desc  : Function to record end time
+  # Input : None
+  # Output: Returns end time of measuring time phase
+  """
+  global end_MT
+  end_MT = time.time()
+  
+
+def printlog(log,phase,request_num,url,start,end,response_time):
+  """
+  # Desc  : Function to Generate per request details in a templog file
+  #         checks all the requests in MT phase are within the start and end time 
+  #         of measuring time phase
+  # Input : log file to collect per request data
+  #         phase of each request(ramp-up, ramp-down, measuring-time)
+  #         request number, URL , start and end time of each request,
+  #         response time of each request
+  # Output: Returns per request details in a templog file
+  """
+  global MT_req
+  if phase =="MT":
+    MT_req = MT_req + 1
+    if ((start >= start_MT and end_MT == 0) or (end_MT > start_MT and end <= end_MT)):
+      print >> log, phase+","+str(request_num)+","+str(url)+","+str(start)+","+str(end)+","+str(response_time)
```

Comment:
Since they are the same, lines 55, 58, and 60 can be extracted out of the if statements and moved to just before the call to flush()